.prowci/plugins.yaml

```diff
@@ -1,8 +1,8 @@
 approve:
 - repos:
   - Azure/aks-engine
-  implicit_self_approve: true
   lgtm_acts_as_approve: true
+  require_self_approval: false

```

Comment:
What behavior does this actually change? No "approve" label required?

Reply:
`implicit_self_approve: true` is deprecated and replaced by `require_self_approve: false`. No behavior change compared to before. The author does not need to approve their own PR.

https://github.com/kubernetes/test-infra/blob/master/prow/plugins/config.go#L233